### PR TITLE
Exclude non-isolated characters in the document name

### PR DESCRIPTION
**Pull request by askonev**

### CODE REVIEW

File: spec/convertion_by_format/check_open_docx_by_screen_spec.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@
       pending 'https://bugzilla.onlyoffice.com/show_bug.cgi?id=32793' if file_path == 'docx/Office Open XML Part 4 - Markup Language Reference.docx'
       pending 'https://bugzilla.onlyoffice.com/show_bug.cgi?id=42324' if file_path == 'docx/SUMMER_APPLICATION_PG_1.docx'
       s3.download_file_by_name('docx/' + File.basename(file_path), './files_tmp')
-      link = "#{StaticData.nginx_url}/#{File.basename(file_path)}"
+      link = "#{StaticData.nginx_url}/#{FileHelper.file_rename(File.basename(file_path))}"
```
Comment (flaminestone) [inline spec/convertion_by_format/check_open_docx_by_screen_spec.rb]: This line is very complex. Can you to divide file renaming and link creating on 2 lines of code

**Review by flaminestone — CHANGES_REQUESTED** [review]

File: spec/convertion_by_format/check_open_docx_by_screen_spec.rb (hunk 1 above)
Comment (ShockwaveNN) [reply spec/convertion_by_format/check_open_docx_by_screen_spec.rb]: Maybe this should be extracted to method

**Comment by ShockwaveNN** [comment]
I think PR body need some examples why this needed and that error cause non-isolated file name

File: spec/convertion_by_format/check_open_docx_by_screen_spec.rb (hunk 1 above)
Comment (askonev) [reply spec/convertion_by_format/check_open_docx_by_screen_spec.rb:16]:
```

Comment (askonev) [reply spec/convertion_by_format/check_open_docx_by_screen_spec.rb]: Isn't such a solution enough?
`tmp_name = FileHelper.file_rename(File.basename(file_path))`
`link = "#{StaticData.nginx_url}/#{tmp_name}"`

Comment (flaminestone) [reply spec/convertion_by_format/check_open_docx_by_screen_spec.rb]: I think, it's enough

**Comment by askonev** [comment]
Document **DRAGON BALLL Z TORRENT 2ª opção.docx**

URL generated:
`http://192.168.1.3:3000/DRAGON BALLL Z TORRENT 2ª opção.docx`

Chrome browser conversion example:
`http://192.168.1.3:3000/DRAGON%20BALLL%20Z%20TORRENT%202%C2%AA%20op%C3%A7%C3%A3o.docx`

Conversion example with normalize function:
`http://192.168.1.3:3000/DRAGON%20BALLL%20Z%20TORRENT%202a%20op%C3%A7%C3%A3o.docx`

There is a problem in converting a character ( ª ) Dex "170"  from ISO-Latin (range 128-255) 

When converting by browser: `%C2%AA`

When converting with normalize: `a`

Similar problem with characters from the list of "unsafe characters" "#"

**Review by flaminestone — APPROVED** [review]

**Review by ShockwaveNN — APPROVED** [review]

**Pull request merged**